cpp/oneapi/dal/algo/kmeans/test/badarg.cpp

```diff
@@ -187,7 +187,7 @@ KMEANS_BADARG_TEST("throws if infer data is empty") {
     REQUIRE_THROWS_AS(infer(kmeans_desc, model, homogen_table{}), domain_error);
 }
 
-KMEANS_BADARG_TEST("throws if assignments are not available") {
+KMEANS_BADARG_TEST("throws if assignments are not set by default") {
```

Comment:
probably it makes sense to remove this case

Reply:
done